curriculum/challenges/english/02-javascript-algorithms-and-data-structures/basic-data-structures/access-an-arrays-contents-using-bracket-notation.md

```diff
@@ -16,47 +16,47 @@ When we define a simple array as seen below, there are 3 items in it:
 let ourArray = ["a", "b", "c"];
 ```
 
-In an array, each array item has an <dfn>index</dfn>. This index doubles as the position of that item in the array, and how you reference it. However, it is important to note, that JavaScript arrays are <dfn>zero-indexed</dfn>, meaning that the first element of an array is actually at the ***zeroth*** position, not the first. In order to retrieve an element from an array we can enclose an index in brackets and append it to the end of an array, or more commonly, to a variable which references an array object. This is known as <dfn>bracket notation</dfn>. For example, if we want to retrieve the `"a"` from `ourArray` and assign it to a variable, we can do so with the following code:
+In an array, each array item has an <dfn>index</dfn>. This index doubles as the position of that item in the array, and how you reference it. However, it is important to note, that JavaScript arrays are <dfn>zero-indexed</dfn>, meaning that the first element of an array is actually at the ***zeroth*** position, not the first. In order to retrieve an element from an array we can enclose an index in brackets and append it to the end of an array, or more commonly, to a variable which references an array object. This is known as <dfn>bracket notation</dfn>. For example, if we want to retrieve the `a` from `ourArray` and assign it to a variable, we can do so with the following code:
 
 ```js
 let ourVariable = ourArray[0];
-// ourVariable equals "a"
 ```
 
+Now `ourVariable` has the value of `a`.
+
 In addition to accessing the value associated with an index, you can also *set* an index to a value using the same notation:
 
 ```js
 ourArray[1] = "not b anymore";
-// ourArray now equals ["a", "not b anymore", "c"];
 ```
 
-Using bracket notation, we have now reset the item at index 1 from `"b"`, to `"not b anymore"`.
+Using bracket notation, we have now reset the item at index 1 from `"b"`, to `"not b anymore"`. Now `ourArray` is `["a", "not b anymore", "c"]`.
 
 # --instructions--
 
 In order to complete this challenge, set the 2nd position (index `1`) of `myArray` to anything you want, besides `"b"`.
 
 # --hints--
 
-`myArray[0]` should be equal to `"a"`
+`myArray[0]` should be equal to `a`
```

Comment:
What about the following hints?  Should they be `to the letter `...`?

Reply:
Letter or string, either one should work.